File: fudi-store/src/test/java/com/romellfudi/sharepreference/resources/ObjectBean.java
package com.romellfudi.sharepreference.resources;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;

import java.util.ArrayList;

/**
 * Created by romelldominguez on 5/15/16.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class ObjectBean {

    private String data;

    private ArrayList<ObjectDetailBean> objectDetailBeans;

    public ObjectBean() {
    }

    public ObjectBean(String data) {
        this.data = data;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public ArrayList<ObjectDetailBean> getObjectDetailBeans() {
        return objectDetailBeans;
    }

    public void setObjectDetailBeans(ArrayList<ObjectDetailBean> objectDetailBeans) {
        this.objectDetailBeans = objectDetailBeans;
    }
}

File: fudi-store-kotlin/src/test/java/com/romellfudi/sharepreference/resources/ObjectBean.kt
/*
 * Copyright (c) 2021. BoostTag E.I.R.L. Romell D.Z.
 * All rights reserved
 * portfolio.romellfudi.com
 */

package com.romellfudi.sharepreference.resources

import com.romellfudi.sharepreference.Fudi
import org.codehaus.jackson.annotate.JsonIgnoreProperties
import java.util.*

/**
 * Created by romelldominguez on 5/15/16.
 */
@Fudi
@JsonIgnoreProperties(ignoreUnknown = true)
data class ObjectBean(var data: String?="", var objectDetailBeans: ArrayList<ObjectDetailBean>?=null) {
}
